src/reynir_correct/errtokenizer.py

```diff
@@ -1709,6 +1724,9 @@ def is_immune(token: CorrectToken) -> bool:
             return True
         if token.txt.isupper():
             # Should not correct all uppercase words
+            if token.error_code and token.error_code in {"Z002"} and at_sentence_start:
```

Comment:
Convert set membership test to a plain equality?

Reply:
Done - thought I would encounter more codes, can always change back if I do.